gammapy/estimators/points/lightcurve.py

```diff
@@ -150,6 +162,20 @@ def run(self, datasets):
                 continue
 
             valid_intervals.append([t_min, t_max])
+            if self.stack_over_time_interval:
+                name = f"timebin_{idx}"
+                dataset_reduced = datasets_to_fit.stack_reduce(name=name)
```

Comment:
This will also stack obervations or event type with different irfs, Is it the intended behavior ?

Reply:
If there are different event types, the user should not put `stack=True`.